website_sale_preset_carts/models/time_frame.py

```diff
@@ -143,11 +143,8 @@ def generate_sale_orders(self):
 
             # we then filter based on suspended date if suspended
             subscribers = customers.filtered(
-                lambda cust: not cust.suspend_cart or (
-                        cust.cart_suspended_from > frame.delivery_date
-                        or cust.cart_suspended_date < frame.delivery_date
-                    )
-                )
+                lambda cust: cust.is_subscribed(frame.delivery_date)
```

Comment:
Just wanted to note that `is_subscribed()` also checks `partner.suspend_cart` and `partner. subscription_id `, so it's a bit more strict then the replaced code, but I think that's a good thing!

Reply:
thanks !